components/BottomNavigation/tests/unit/MDCBottomNavigationBarControllerTests.m

```diff
@@ -313,7 +368,56 @@ - (void)testNonSelectedViewControllersHaveViewsLazilyLoaded {
   XCTAssertFalse(childViewController2.viewLoaded);
 }
 
-#pragma mark - MDCBottomNavigationBarControllerDelegate Methods
+- (void)testEncodesChildViewControllersWhenPreservingRestorationState {
+  // Given
+  UIViewController *childViewController1 = [[UIViewController alloc] init];
+  childViewController1.restorationIdentifier = @"vc1";
+
+  UIViewController *childViewController2 = [[UIViewController alloc] init];
+  childViewController2.restorationIdentifier = @"vc2";
+
+  self.bottomNavigationBarController.viewControllers =
+      @[ childViewController1, childViewController2 ];
+  self.bottomNavigationBarController.viewControllers[0].restorationIdentifier = @"vc1";
+  self.bottomNavigationBarController.viewControllers[1].restorationIdentifier = @"vc2";
```

Comment:
Is this duplicating the property assignments above or am I missing something?

Reply:
Done.